<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuickAddCapacitorDialog</class>
 <widget class="QDialog" name="QuickAddCapacitorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>324</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add capacitor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../extras.qrc">
    <normaloff>:/parts/extras/category/ceramic_16x16.png</normaloff>:/parts/extras/category/ceramic_16x16.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="KMessageWidget" name="messageWidget" native="true"/>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QWidget" name="widget" native="true"/>
     </item>
     <item row="0" column="1">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>8</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Capacitance</string>
       </property>
       <property name="buddy">
        <cstring>capacitanceValueLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="capacitanceValueLineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Voltage rating:</string>
       </property>
       <property name="buddy">
        <cstring>voltageRatingLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="voltageRatingLineEdit">
       <property name="inputMethodHints">
        <set>Qt::ImhNone</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Tolerance:</string>
       </property>
       <property name="buddy">
        <cstring>toleranceLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="toleranceLineEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Quantity:</string>
       </property>
       <property name="buddy">
        <cstring>quantitySpinBox</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="quantitySpinBox"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Package:</string>
       </property>
       <property name="buddy">
        <cstring>partPackageComboBox</cstring>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="RelationalComboBox" name="partPackageComboBox"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Category:</string>
       </property>
       <property name="buddy">
        <cstring>partCategoryComboBox</cstring>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="TreeViewComboBox" name="partCategoryComboBox"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Storage:</string>
       </property>
       <property name="buddy">
        <cstring>partStorageComboBox</cstring>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="TreeViewComboBox" name="partStorageComboBox"/>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Condition:</string>
       </property>
       <property name="buddy">
        <cstring>partConditionComboBox</cstring>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="RelationalComboBox" name="partConditionComboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::MinimumExpanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Reset|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TreeViewComboBox</class>
   <extends>QComboBox</extends>
   <header>widgets/treeviewcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>RelationalComboBox</class>
   <extends>QComboBox</extends>
   <header>widgets/relationalcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KMessageWidget</class>
   <extends>QWidget</extends>
   <header>widgets/kmessagewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../extras.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QuickAddCapacitorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>404</x>
     <y>295</y>
    </hint>
    <hint type="destinationlabel">
     <x>404</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
